packages/platform/src/setup/setupMCPChatHandler.ts

```diff
@@ -0,0 +1,78 @@
+import { UnchainedCore } from '@unchainedshop/core';
+import { Experimental_StdioMCPTransport as StdioMCPTransport } from 'ai/mcp-stdio';
+import { Request, RequestHandler, Response } from 'express';
```

Comment:
This handler is express-specific and has to be part of the handler folder in the api package. platform can not have any direct dependency to a webserver lik express/fastify.